webui/react/src/components/ManageGroupsModal.tsx

```diff
@@ -103,29 +83,39 @@ const useModalManageGroups = ({ groups, user }: ModalProps): ModalHooks => {
     [form, user],
   );
 
-  const fetchUserGroups = useCallback(async () => {
-    try {
-      const response = await getGroups({ userId: user.id });
-      const groupIds = response.groups?.map((ug) => ug.group.groupId);
-      return groupIds;
-    } catch (e) {
-      handleError(e, { publicSubject: "Unable to fetch this user's groups." });
-    }
-  }, [user.id]);
+  if (!rbacEnabled) {
+    return null;
+  }
 
-  const modalOpen = useCallback(async () => {
-    const userGroupIds = await fetchUserGroups();
-    openOrUpdate({
-      closable: true,
-      content: <ModalForm form={form} groupOptions={groups} userGroupIds={userGroupIds} />,
-      icon: null,
-      okText: 'Save',
-      onOk: () => handleOk(userGroupIds),
-      title: <h5>Manage Groups</h5>,
-      width: 520,
-    });
-  }, [form, handleOk, openOrUpdate, fetchUserGroups, groups]);
-  return { modalOpen, ...modalHook };
+  return (
+    <Modal
+      cancel
+      size="small"
+      submit={{
```

Comment:
the value is not set correctly after editing and saving 

https://user-images.githubusercontent.com/109113616/229241302-166b018f-1324-4979-a239-d13ce73f46a5.mov

Reply:
i think this might be working as intended -- the "Add Users" dropdown doesn't display selected users, it just shows/hides users in the list that appears on click. This could perhaps be adjusted to be less confusing, but I think that's a separate design/UX issue.

https://user-images.githubusercontent.com/97752292/229578509-2a85f16e-5c38-46b6-9077-a6a79fe03279.mov

Reply:
actually nvm this should be displaying the selected users just like every other multiple mode `Select`. but it wasn't working in the previous version of the modal either. looking into it.

Reply:
so does that mean you fix the issue or not in this PR? 
i still see the issue. If its not a plan to fix the issue in this PR, just ignore this.

https://user-images.githubusercontent.com/109113616/229883132-4fc72c03-1af2-473b-a471-6f9bb0322ed4.mov